examples/quickstart_etl/README.md

```diff
@@ -0,0 +1,208 @@
+# Dagster starter kit
+
+This example is a starter kit for building a daily ETL pipeline. It showcases Dagster's basic features such as **[Assets](https://docs.dagster.io/concepts/assets/software-defined-assets)** and **[Schedules](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules)**. It also follows Dagster's recommendations on [using environment variables and secrets](https://docs.dagster.io/guides/dagster/using-environment-variables-and-secrets) to ensure the project is production-ready.
+
+This project is scaffolded with [`dagster project scaffold`](https://docs.dagster.io/getting-started/create-new-project).
+
+
+## Introduction
+
+<p align="center">
+    <img height="500" src="https://user-images.githubusercontent.com/4531914/200668814-2c0be35e-25eb-4555-a3a5-f7e52ff792f4.png">
+</p>
+
+
+In this project, we're building an analytical pipeline that:
```

Comment:
I think it's worth adding some words explaining the point of this pipeline - e.g. "monitor the popularity of open source software projects" or whatever it is.